libtelemetry/src/test/java/com/mapbox/android/telemetry/MapboxTelemetryTest.java

```diff
@@ -164,6 +187,49 @@ public void checksOnEventReceivedPushNotCalledWhenTelemetryDisabled() throws Exc
     verify(mockedEventsQueue, never()).push(eq(mockedEvent));
   }
 
+  @Test
+  public void checksPush() throws Exception {
```

Comment:
This test is doing the same exact thing as `checksOnEventReceivedPushNotCalledWhenTelemetryDisabled` and should be removed. It doesn't add any value.